src/index.js

```diff
@@ -41,9 +41,9 @@ function loader(...args) {
   // this.fs can be undefined
   // e.g when using the thread-loader
   // fallback to the fs module
-  const stat = this.fs ? this.fs.stat : fs.stat;
+  const tempFs = this.fs || fs;
```

Comment:
```suggestion
  const FS = this.fs || fs;
```

Reply:
@michael-ciniawsky `FS`? Why in uppercase? Why not `usingFs` or something related?